pkg/dagger.io/dagger/core/exec.cue

```diff
@@ -44,6 +44,48 @@ import "dagger.io/dagger"
 	exit: int & 0
 }
 
+#AsyncExec: {
```

Comment:
Not necessary, but we could use another (probably hidden) definition to pull out the commonalities between `#Exec` and `#AsyncExec`.